weave-js/src/components/PagePanelComponents/Home/Browse3/grid/sort.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  GridSortDirection,
+  GridSortItem,
+  GridSortModel,
+} from '@mui/x-data-grid-pro';
+import _ from 'lodash';
+
+const isValidSortDirection = (obj: any): obj is GridSortDirection => {
+  return obj === 'asc' || obj === 'desc' || obj === null || obj === undefined;
+};
+
+const isValidSortItem = (obj: any): obj is GridSortItem => {
+  if (!_.isPlainObject(obj)) {
```

Comment:
what does this do?

Reply:
Checks whether the passed in thing is a JavaScript object like `{"key": "value"}` - `_.isObject` would also return true for things like arrays or functions.

Reply:
ah, right, thx